src/components/Editor/Preview/ReactPopup.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+import React, { Component } from "react";
+
+import "./ReactPopup.css";
+
+class ReactPopup extends Component {
+  props: {
+    roots: Array<Object>
+  };
+
+  renderComponentAttributes(root: Object) {
+    if (!("preview" in root.contents.value)) {
+      return <span className="none">none</span>;
+    }
+    const attributes = root.contents.value.preview.ownProperties;
```

Comment:
I don't know if this is the best way to grab properties... `ObjectInspector` uses `const getObjectProperties = id => loadedObjects[id];` but I couldn't get that to work :/ Any insight is appreciated ;)